src/cmd_vel_mux.cpp

```diff
@@ -166,8 +192,15 @@ void CmdVelMux::configureFromParameters(const std::vector<std::string> & names,
   RCLCPP_INFO(get_logger(), "CmdVelMux : (re)configured");
 }
 
-void CmdVelMux::cmdVelCallback(const std::shared_ptr<geometry_msgs::msg::Twist> msg, unsigned int idx)
+void CmdVelMux::cmdVelCallback(const std::shared_ptr<geometry_msgs::msg::Twist> msg, std::string name)
 {
+  unsigned int idx = VACANT;
+  for (unsigned int i = 0; i < list_.size(); i++)
+  {
+    if (list_[i]->name_ == name) idx = i;
+  }
```

Comment:
Since this happens on every cmd_vel callback (potentially 100's of times per second), this is going to be too slow, and will get slower as we add more inputs.  I *think* we should be able to find this stuff by index still, but if not we need to switch to something else like a `map`.

Same goes for the `timerCallback` below, though it is somewhat less concerning there.